config/routes.rb

```diff
@@ -38,9 +38,11 @@
   get '/register', to: 'users#new'
   post '/register', to: 'users#create'
 
-  get "/profile", to: 'users#show'
+  get '/profile', to: 'users#show'
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
 
   get '/merchant/dashboard', to: 'dashboards#show'
+
+  get '/admin/dashboard', to: 'dashboards#show'
```

Comment:
I would assume these shouldn't be the same dashboards. Here is where we might need to namespace.